test/seq/runtests.jl

```diff
@@ -10,6 +10,7 @@ end
 import Compat
 using Bio.Seq,
     BufferedStreams,
+    StatsBase,
```

Comment:
Forgive me if I'm wrong, but I can't see where  `StatsBase` is still used? I assume this was included for the now-commented out `cdf(Binomial(...))` calls?